host-configs/lc-builds/toss3/hip_4_5_link_X.cmake

```diff
@@ -0,0 +1,32 @@
+###############################################################################
+# Copyright (c) 2016-22, Lawrence Livermore National Security, LLC
+# and RAJA project contributors. See the RAJA/LICENSE file for details.
+#
+# SPDX-License-Identifier: (BSD-3-Clause)
+###############################################################################
+
+set(RAJA_COMPILER "RAJA_COMPILER_CLANG" CACHE STRING "")
+
+set(CMAKE_CXX_FLAGS_RELEASE "-O2" CACHE STRING "")
+set(CMAKE_CXX_FLAGS_RELWITHDEBINFO "-O2 -g" CACHE STRING "")
+set(CMAKE_CXX_FLAGS_DEBUG "-O0 -g" CACHE STRING "")
+
+set(HIP_COMMON_OPT_FLAGS )
+set(HIP_COMMON_DEBUG_FLAGS)
+set(HOST_OPT_FLAGS)
+
+if(DEFINED ROCM_ROOT_DIR)
+  set(CMAKE_EXE_LINKER_FLAGS "-Wl,--disable-new-dtags -L${ROCM_ROOT_DIR}/hip/lib -L${ROCM_ROOT_DIR}/lib -L${ROCM_ROOT_DIR}/lib64 -Wl,-rpath,${ROCM_ROOT_DIR}/hip/lib:${ROCM_ROOT_DIR}/lib:${ROCM_ROOT_DIR}/lib64 -lamdhip64 -lhsakmt -lhsa-runtime64 -lamd_comgr" CACHE PATH "")
```

Comment:
Interesting . . . we need an rpath even though we've already set `ROCM_ROOT_DIR`?

Reply:
I think -L paths are checked at link time and rpaths are checked at runtime, even for shared libs.